src/sentry/remote_config/storage.py

```diff
@@ -105,5 +95,65 @@ def _make_storage_config(self) -> dict | None:
             return None
 
 
-def make_storage(key: Project):
-    return StorageBackend(key)
+class ConfigurationBackend:
```

Comment:
Its an abstraction to manage fetching the configs.  I have no idea how long this will live.  Its not intended to be permanent.  We can document it later.